arangod/Aql/Query.cpp

```diff
@@ -1462,7 +1462,7 @@ bool Query::canUseQueryCache() const {
 
 ErrorCode Query::resultCode() const noexcept {
   // never return negative value from here
-  return _resultCode.value_or(TRI_ERROR_NO_ERROR);
+  return _resultCode.load().value_or(TRI_ERROR_NO_ERROR);
```

Comment:
Is it ok to use memory order `std::memory_order_relaxed` for this load?

Reply:
I'm really unsure about memory ordering. As it is now it is at least correct, do you think that using `std::memory_order_relaxed` is really relevant here?

Reply:
I don't think it is relevant here, because there is just one call site.
So feel free to ignore it.